.github/workflows/continuous-deployment.yml

```diff
@@ -139,6 +139,26 @@ jobs:
       - name: Run pa11y Test
         run: ../ops/helper-scripts/accessibility-test.sh
 
+  sast-scan-and-upload:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Package source code for scan
+        run: zip -r cams.zip . -i "./backend/*" -i "./user-interface/*"
+
+      - name: Veracode Upload And Scan
+        uses: veracode/veracode-uploadandscan-action@0.2.6
+        with:
+          appname: 'CAMS'
+          createprofile: false
+          filepath: 'cams.zip'
+          vid: '${{ secrets.VERACODE_API_ID }}'
+          vkey: '${{ secrets.VERACODE_API_KEY }}'
+          criticality: 'Medium'
```

Comment:
I might prefer this to go into its own workflow, but I'm sure here is fine for now at least.